### DISCUSSION TITLE
iOS - Where to find a downloaded file

### DISCUSSION THREAD

**Discussion by zvitek** [ionic-team/capacitor#4057] [Q&A]
Hello, 
I have this script to download a file to phone.

```js
Filesystem.writeFile({
    path:      fileName,
    data:      base64data,
    directory: FilesystemDirectory.Documents
}).then(() => {
    Filesystem.getUri({
        directory: FilesystemDirectory.Documents,
        path:      fileName
    }).then((getUriResult) => {
        const path = getUriResult.uri;
        console.log(path);
    }, (error) => {
        console.log(error);
    });
},
err => console.log(err));
```

In android everything works good and I can found the downloaded file in file manager. 
In iOS everything works too except to find a downloaded file.

The console returns the path of saved file but I have no idea where I can to find a file in Files application or somewhere else.
```
{"uri":"file:\/\/\/Users\/***\/Library\/Developer\/CoreSimulator\/Devices\/80124C86-7581-46EF-84B3-18763A8EC0D8\/data\/Containers\/Data\/Application\/D6DA4041-9B08-4168-BB34-696BB415D5BF\/Documents\/file.pdf"}
```
I tried Data and Document folder, but result was same.
Can you tell me where can be a problem or if I doing something wrong. 
It's possible to save a file to Files application?

Thank you for help

I'm using Capacitor with a Polymer 3 app.

Capacitor version 2.4.1
Node version 12.20.1

**Comment by jcesarmobile** [comment]
Try adding this in your info.plist

```
<key>UIFileSharingEnabled</key>
<true/>
```
https://developer.apple.com/documentation/bundleresources/information_property_list/uifilesharingenabled

**Reply by zvitek** [reply]
Thank you, but unfortunately, it didn't help :/

**Reply by jcesarmobile** [reply]
wops, you also need


```
<key>LSSupportsOpeningDocumentsInPlace</key>
<true/>
```

**Reply by bipoza** [reply]
Thank you!
I have used it in combination with [http plugin to download files](https://github.com/capacitor-community/http/blob/master/example/src/components/app-home/app-home.tsx#L208). With this configuration in IOS the feeling is completely native. This creates a folder with the application name and icon in the Files application. This is wonderful ;)

**Reply by Frank608** [reply]
Saved my life 😅

**Comment by santiconte** [comment]
Hello @zvitek, were you able to find a solution for this? The options proposed by jcesarmobile would not be giving me an effect. Greetings

**Reply by umeshgiri8** [reply]
hi @santiconte it did not show up in the Files app of the simulator but it worked on the actual device for me
added these in info.plist
``
<key>UIFileSharingEnabled</key>
<true/>
<key>LSSupportsOpeningDocumentsInPlace</key>
<true/>
``

**Reply by santiconte** [reply]
Yes, exactly the same thing happened to me.
